workflows/prognostic_c48_run/runtime/steppers/machine_learning.py

```diff
@@ -189,8 +189,10 @@ def __call__(self, time, state):
         for key, value in prediction.items():
             if is_state_update_variable(key, state):
                 state_updates[key] = value
-            else:
+            elif is_tendency_variable(key):
```

Comment:
It's good to have this check for whether the tendency corresponds to. With the new `else` statement that catches everything that isn't applied as an update, there's a new way for a ML model to silently fail by inadvertently using a real update as an intermediate variable if the output names are different or if the renaming adapter is misconfigured. Could you add a logging statement to record if model predictions are put in the diagnostics? I think the tendency and state updates are logged but I think it'll be more direct to diagnose a possible issue if the diagnostic predictions are also logged.

Reply:
Good idea, yeah, that could be awkward to debug otherwise.  I added logging messages within the time loop near where the other updates are logged -- let me know if those look good.

Reply:
Thanks, they LGTM